GTG/core/requester.py

```diff
@@ -172,17 +172,41 @@ def new_tag(self, tagname):
         """
         return self.ds.new_tag(tagname)
 
-    def new_search_tag(self, name, query):
+    def new_search_tag(self, query):
         """
         Create a new search tag from search query
 
         Note: this modifies the datastore.
 
-        @param name:  name of the new search tag
         @param query: Query will be parsed using search parser
-        @return:      new tag
+        @return:      Name of the tag
```

Comment:
`tag id` might be more appropriate (codebase always calls it either `task_id`, `tag_id`, or `tid`)